src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { ChangeEvent } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+
+type AuthorSearchInputProps = {
+  text: string;
+  onChange: (e: ChangeEvent<HTMLInputElement>) => void;
+};
+
+// TODO: when SearchInput is ported to next js this style
+// (also found in PostSearchInput.tsx) can be adequately ported as well
+// i.e. by moving the below style to the SearchInput component
```

Comment:
I think it'd be better to have this comment in the affected issues instead of here so we won't have to hope that the people implementing the other components see this.

Reply:
Sounds good to me, I'll remove it and place it in the appropriate issue. Thanks Josue!